microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -34,6 +37,15 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `POST` | `/products/{productID}/submissions/draft/package` |
```

Comment:
```suggestion
| `POST` | `/products/$productID/submissions/draft/package` |
```
global: @yvsubhash - 
change {operationID} to  $operationID  ?
change {productID}   to  $productID    ?
change {operationID} to  $operationID  ?

consistency pass - live docs use 3-4 different notations for seemingly same variable

Reply:
Ans: We can standardize "productID" and "operationID" var notation to same format consistently throughout using .md & ref .md